packages/xcoobee-cookie-kit-react/src/xck-react-theme-overlay.scss

```diff
@@ -69,6 +69,10 @@ $duration: .2s;
     opacity: 0;
   }
 
+  &.hide {
```

Comment:
Style rules like these are sometimes known as utility classes. Utility classes always use `!important` with their properties. This is because if the utility class exists on an element, then its styles should apply. But because utility classes inherently have a low specificity, sometimes the properties from other higher specificity rules override the utility class rules.

See https://getbootstrap.com/docs/4.3/utilities/display/ for how bootstrap 4 names their utility classes. I recommend that we use `d-none`.

Note: The utility classes need not be nested. They should be at the top-level.

Because we have two SASS style sheets that shares some common rules, we can extract those into a partial (a SASS file prefixed with `_`).

Please let me know if you have any questions about this.

Reply:
Ok.
The only thing – besides this class we have something more related to both themes, so I think we may need a separate ticket for this stuff to fully reorganize styles files.

Reply:
Sounds good.